nft_ingester/src/config.rs

```diff
@@ -66,20 +76,31 @@ impl IngesterConfig {
         mc
     }
 
-    pub fn get_account_stream_worker_count(&self) -> u32 {
-        self.account_stream_worker_count.unwrap_or(2)
+    pub fn get_worker_config(&self) -> Vec<WorkerConfig> {
+        return if let Some(wc) = &self.worker_config {
+            wc.to_vec()
+        } else {
+            vec![
+                WorkerConfig {
+                    stream_name: "ACC".to_string(),
+                    worker_count: 2,
+                    worker_type: WorkerType::Account,
+                },
+                WorkerConfig {
+                    stream_name: "TXN".to_string(),
+                    worker_count: 2,
+                    worker_type: WorkerType::Transaction,
+                },
+            ]
```

Comment:
If worker configs are tied to the stream shouldn't they be derived from the worker type to avoid misconfiguration?

Also related, what are folks thoughts on using clap for this project and having different command entry points for the different worker types instead of using an environment variable. Idea is to provide more structure to the code base and promote providers running the workers using separate processes as suggested in the production guide.

Reply:
> different command entry points for the different worker types instead of using an environment variable

I hope that we will test and migrate to the new ingester soon :slightly_smiling_face:  it's possible to define streams to consume in config with a lot of options https://github.com/rpcpool/digital-asset-rpc-infrastructure/blob/pt-ingester2/nft_ingester2/config-ingester.yml